frontend/app_flowy/lib/workspace/presentation/stack_page/home_stack.dart

```diff
@@ -29,7 +29,8 @@ class HomeStack extends StatelessWidget {
         getIt<HomeStackManager>().stackTopBar(),
         Expanded(
           child: Container(
-            color: Colors.white,
+            //Rewrote to change with light and dark mode
```

Comment:
please remove unneeded comment (this comment only helps with this commit. It won't help future developers. Get it?)